include/sc2api/sc2_unit.h

```diff
@@ -218,8 +218,11 @@ class Unit {
 };
 
 typedef std::vector<const Unit*> Units;
+typedef std::vector<Tag> Tags;
 typedef std::unordered_map<Tag, size_t> UnitIdxMap;
 
+Tags GetTagsFromUnits(const Units& units);
```

Comment:
Please rename this function to `ConvertToTags`. This is minor style issue, the new name is more consistent with other parts of the code.